service/src/main/java/bio/terra/cbas/controllers/ControllerUtils.java

```diff
@@ -0,0 +1,31 @@
+package bio.terra.cbas.controllers;
+
+import bio.terra.cbas.dependencies.sam.SamService;
+import bio.terra.cbas.util.BearerTokenFilter;
+import java.util.Optional;
+import org.broadinstitute.dsde.workbench.client.sam.model.UserStatusInfo;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.web.context.request.RequestAttributes;
+import org.springframework.web.context.request.RequestContextHolder;
+
+@Component
+public class ControllerUtils {
```

Comment:
Clarifying question: I am wondering if these here can be moved inside SamService as they seem dependent on it? Or is there a reason that they should be in a separate class? 🤔

Reply:
@cjllanwarne does this approach make sense over the `ControllerUtils` static methods suggested above? (Could just get rid of ControllerUtils)